lxd-agent/main_agent.go

```diff
@@ -218,7 +199,7 @@ func (c *cmdAgent) Run(cmd *cobra.Command, args []string) error {
 
 // startStatusNotifier sends status of agent to vserial ring buffer every 10s or when context is done.
 // Returns a function that can be used to update the running status to STOPPED in the ring buffer.
-func (c *cmdAgent) startStatusNotifier(ctx context.Context) context.CancelFunc {
+func (c *cmdAgent) startStatusNotifier(ctx context.Context, chConnected chan struct{}) context.CancelFunc {
```

Comment:
As this function only receives from `chConnected` it would be nice to indicate this in the argument spec, e.g. `chConnected <-chan struct{}`. This helps clarify what is read/writing to the channel.

See https://golangbyexample.com/channel-function-argument-go/